src/lib/core/testing/dispatch-events.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  createFakeEvent,
+  createKeyboardEvent,
+  createMouseEvent,
+  createTransitionEndEvent
+} from './event-objects';
+
+/** Shorthand to dispatch a fake event on a specified node. */
+export function dispatchFakeEvent(node: Node, eventName: string) {
```

Comment:
I see, I was interpreting "fake" as "not caused by user interaction". I guess this is ok then